src/components/Header.js

```diff
@@ -10,10 +10,20 @@ export default class Header extends React.Component {
   }
 
   render(){
+
+    // Determine which login button to display
+    let loginButton;
+    if(this.props.childProps.isAuthenticated === 'true'){
```

Comment:
Here was the issue... `isAuthenticated` is stored as a string, not as a boolean, so `if(this.props.childProps.isAuthenticated)` will always evaluate as true.